src/API/Common/Event/Dispatcher.php

```diff
@@ -5,20 +5,16 @@
 namespace OpenTelemetry\API\Common\Event;
 
 use CloudEvents\V1\CloudEventInterface;
-use OpenTelemetry\Context\Context;
-use OpenTelemetry\Context\ContextKey;
 
 class Dispatcher implements DispatcherInterface
 {
-    private static ?ContextKey $key = null;
+    private static ?self $root = null;
     /** @var array<string, array<int, array<callable>>> */
     private array $listeners = [];
 
-    public static function getInstance(): self
+    public static function getRoot(): self
```

Comment:
I chose `root` because it sounds less singleton-y, since you can in fact have as many dispatchers as you wish. This is just the default one.